server/entitymanager/entitymanager.go

```diff
@@ -0,0 +1,31 @@
+// Package entitymanager is an in-memory implementation of an entity manager that models an organization's inventory.
+package entitymanager
+
+import (
+	"github.com/openconfig/bootz/proto/bootz"
+	"github.com/openconfig/bootz/server/service"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type InMemoryEntityManager struct{}
+
```

Comment:
Can we add docstrings for each of these exported methods? I think probably they would be best where the `interface` is defined and then you can just mark them as needed to implement the interface here?

Reply:
not sure why the lint didn't complain on this